yucca/pipeline/task_conversion/Task012_BraTS21.py

```diff
@@ -89,9 +89,12 @@ def convert(path: str, subdir: str = "brats21/training_data"):
         target_imagesTr,
         target_imagesTs,
         ("FLAIR", "T1", "T1CE", "T2"),
-        labels={0: "background", 1: "necrotic tumor core", 2: "peritumoral edematous/invaded tissue", 3: "GD-enhancing tumor"},
-        regions_in_order=[[1, 2, 3], [1, 3], [3]],
-        regions_labeled=[1, 2, 3],
+        labels={0: "BG", 1: "NCR", 2: "ED", 3: "ET"},
+        regions={
+            "WT": {"priority": 3, "labels": ["ET", "NCR", "ED"]},
```

Comment:
maybe it could become             "WT": {"priority": 3, "labels": [1, 2, 3]},
so they match the keys instead of the values

Reply:
My reasoning is this:

1. I wanted something where it is easy to translate a challenge submission to a region specification.
2. Label values are dependent on the task conversion file in e.g. BraTS.

I would say either are equally fragile. However, the asserts here, make sure that the long string matching will never be a problem, since this will be caught when task is converted: 

https://github.com/Sllambias/yucca/blob/4b11081affe14aa806f8fe6339f0aed044b96a1e/yucca/pipeline/task_conversion/utils.py#L117-L132